frontend/src/views/score.js

```diff
@@ -32,29 +67,111 @@ function Score() {
+    alignItems: "center"
+  };
 
-  function handleChange(event) {
-    event.preventDefault();
-    setUsername(event.target.value);
-  }
+  const titleCss = {
+    fontWeight: "bolder",
+    fontSize: 30,
+    textAlign: "center",
+    marginBottom: "5%",
+    marginTop: "20%"
+  };
+
+  const bottomButtonsCss = {
+    width: "100%",
+    display: "flex",
+    justifyContent: "space-around",
+    margin: "20% 0"
+  };
+
+  const scoresCss = {
+    display: "flex",
+    flexDirection: "column",
+    width: "max-content",
+    margin: 0,
+    alignItems: "center",
+    fontSize: 14
+  };
 
   return (
-    <form onSubmit={handleSubmit}>
-      <input
-        css={styleInputCss}
-        type="text"
-        autoComplete="off"
-        name="username"
-        aria-label="Enter your name"
-        onChange={handleChange}
-      />
-      <Button type="submit">SAVE MY SCORE</Button>
-    </form>
+    <main css={{ width: "100%" }}>
+      <h1 css={titleCss}>Final Score</h1>
+      <div
+        css={{
+          marginBottom: "40%",
+          width: "100%",
+          display: "flex",
+          justifyContent: "space-around"
+        }}
+      >
+        <section css={scoresCss}>
+          <span css={{ width: "max-content" }}>TOTAL REVENUE</span>
+          <span css={{ width: "max-content", fontSize: 18 }}>$ {total}</span>
+        </section>
+        <section css={scoresCss}>
+          <span css={{ width: "max-content" }}>ACCURACY (C.R.)</span>
+          <span css={{ width: "max-content", fontSize: 18 }}>{correct}/7</span>
+        </section>
+      </div>
+      <form
+        onSubmit={handleSubmit}
+        css={{
+          display: "flex",
+          flexDirection: "column",
+          textAlign: "center"
+        }}
+      >
+        <section
+          css={{
+            width: "80%",
+            textAlign: "center",
+            margin: "0 auto",
+            marginBottom: "5%"
+          }}
+        >
+          Save your score and see your position in the leaderboard
+        </section>
+        <label htmlFor="username" css={{ margin: "0 auto" }}>
+          Name
+        </label>
+        <input
+          css={styleInputCss}
+          type="text"
+          autoComplete="off"
+          name="username"
+          id="username"
+          aria-label="Enter your name"
+          onChange={handleChange}
+          required
+        />
+        <Button type="submit">SAVE MY SCORE</Button>
+      </form>
+      <div css={bottomButtonsCss}>
+        <div css={buttonLabelCss}>
+          <Button
+            css={{ width: 50, height: 50, borderRadius: "50%" }}
+            onClick={handleAgain}
+          >
+            <i class="fas fa-redo fa-2x" />
+          </Button>
+          <span css={{ flexShrink: 1 }}>Play again</span>
+        </div>
+        <div css={buttonLabelCss}>
+          <Button
+            css={{ width: 50, height: 50, borderRadius: "50%" }}
+            onClick={handleLeaderboard}
```

Comment:
Hi Vale. What do you think about using `Link` instead of calling a method that only do a `navigate`. And if you want use the `Button` component you could do the following:
```javascript
<Link to="/path">
  <Button>Name button</Button>
</Link>
```

Reply:
Didn't thought of it, imma do it now